spec/serializers/api/v2/serializable_contact_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+describe API::V2::SerializableContact do
+  let(:contact) { build(:contact) }
+  let(:resource) { described_class.new(object: contact) }
+
+  it "sets type to contacts" do
+    expect(resource.jsonapi_type).to eq(:contacts)
+  end
+
+  subject { JSON.parse(resource.as_jsonapi.to_json) }
+
+  it { should have_type "contacts" }
```

Comment:
We should (no pun intended) use `is_expected.to` rather than `should` here.

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/1579/commits/d1a210cdbf10e1cca1e2c95897167a09da4a88ac